src/catalog/src/memory/objects.rs

```diff
@@ -575,10 +575,16 @@ pub enum DataSourceDesc {
     },
     /// This source receives its data from the identified ingestion,
     /// specifically the output identified by `external_reference`.
+    /// N.B. that `external_reference` should not be used to identify
```

Comment:
Just to confirm, do we nevertheless keep this piece of data around right?

Reply:
Yes it's stored on the statement itself and populated onto this memory object for use in system tables, but otherwise isn't stored / sent anywhere